scanner/src/main/kotlin/scanners/fossid/FossId.kt

```diff
@@ -169,10 +182,14 @@ class FossId(
                 val revision = pkg.vcsProcessed.revision.ifEmpty { "HEAD" }
                 val projectCode = convertGitUrlToProjectName(url)
 
-                log.info { "Creating project '$projectCode' ..." }
+                val project = getProject(projectCode)
```

Comment:
Nit: This variable could be inlined. Or are you planning to make more use of it later?